python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -34,28 +34,28 @@
 
 # mypy: ignore-errors
 import os
+from time import time
 
 import torch
 import torch.nn as nn
 import torch.nn.functional as F
+from torchmetrics import Accuracy
 import torchvision
-from pytorch_lightning import LightningModule, Trainer, seed_everything
+from pl_bolts.datamodules import CIFAR10DataModule
+from pl_bolts.transforms.dataset_normalizations import cifar10_normalization
+from pytorch_lightning import LightningModule, seed_everything
 from pytorch_lightning.callbacks import LearningRateMonitor
 from pytorch_lightning.loggers import TensorBoardLogger
 from torch.optim.lr_scheduler import OneCycleLR
-from torchmetrics.functional import accuracy
-
-
-from pl_bolts.datamodules import CIFAR10DataModule
-from pl_bolts.transforms.dataset_normalizations import cifar10_normalization
+from torchmetrics import Accuracy
 
+from bigdl.nano.pytorch.trainer import Trainer
 
 seed_everything(7)
 
-PATH_DATASETS = os.environ.get("PATH_DATASETS", ".")
+PATH_DATASETS = os.path.dirname(os.path.abspath(__file__))
 BATCH_SIZE = 64
-NUM_WORKERS = int(os.cpu_count() / 2)
-
+NUM_WORKERS = 0  # Multi-thread run sometimes raise quantization error
```

Comment:
Oh really? Could you raise a issue for record? since I'm working on num_workers recently.